### PR TITLE
Fix integration tests

### PR DESCRIPTION
**Pull request by armenzg**

### CODE REVIEW

**Comment by ahal** [comment]
The fact that `reftest` flavors are missing manifests is either a legit regression or is exposing some deeper flaw in our test. Either way we should investigate further.

**Comment by marco-c** [comment]
I see the test was skipped in the latest cron run: https://travis-ci.org/github/mozilla/mozci/builds/669726057.

**Comment by marco-c** [comment]
The last build where we run them was https://travis-ci.org/github/mozilla/mozci/builds/663523324, the first where we didn't was https://travis-ci.org/github/mozilla/mozci/builds/663967977. So #91 didn't work as expected.

File: tests/test_integration.py [hunk 1]
```diff
@@ -17,15 +17,20 @@ def test_missing_manifests():
...
+        "reftest",
         "talos",
+        "test-verify-wpt",
+        "web-platform-tests",
+        "web-platform-tests-crashtest",
+        "web-platform-tests-crashtests",
+        "web-platform-tests-reftest",
         "web-platform-tests-reftests",
         "web-platform-tests-wdspec",
-        "web-platform-tests-crashtests",
```
Comment (marco-c) [inline tests/test_integration.py:24]: The test passes for me by just adding `web-platform-tests-crashtest` and `web-platform-tests-reftest`

**Comment by armenzg** [comment]
Do you want me to reduce the list and get that landed? or are you still investigating as to why this started happening?

**Comment by marco-c** [comment]
> Do you want me to reduce the list and get that landed?

Yep!

>  or are you still investigating as to why this started happening?

I think it was changed on purpose by @ahal .

File: tests/test_integration.py (hunk 1 above)
Comment (marco-c) [reply tests/test_integration.py:24]: Is it passing this way? Or are there both `web-platform-tests-reftests` and `web-platform-tests-reftest`?

Comment (armenzg) [reply tests/test_integration.py:24]: It seems that only the singular ones are the ones passing.

Comment (ahal) [reply tests/test_integration.py:24]: This makes sense, Edwin and I recently changed the labels to remove the plural (to match what the WPT harness actually calls them). Sorry I didn't think of this sooner.. for some reason I thought this was happening to normal reftest/crashtest.

Comment (marco-c) [reply tests/test_integration.py:24]: Yeah I was worried there could still be some of the old ones in the results from the query, but I guess the change has happened enough time ago.

**Pull request merged**